ydb/public/api/grpc/ydb_tablet_v1.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
I suppose it's better to move this file into draft folder.

Reply:
Done